.github/workflows/install-valkey/action.yml

```
@@ -62,6 +62,7 @@ runs:
             echo 'export PATH=/usr/local/bin:$PATH' >>~/.bash_profile
 
         - name: Verify Valkey installation and symlinks
+          if: ${{ !contains(inputs.engine-version, '-rc1') }}
```

it's too specific. change to
```suggestion
          if: ${{ !contains(inputs.engine-version, '-rc') }}
```